plugins/org.eclipse.fordiac.ide.model/src-gen/org/eclipse/fordiac/ide/model/libraryElement/impl/EventImpl.java

```diff
@@ -337,10 +337,11 @@ public DataType basicGetType() {
 	/**
 	 * <!-- begin-user-doc -->
 	 * <!-- end-user-doc -->
-	 * @generated
+	 * @generated NOT
 	 */
 	@Override
 	public void setType(DataType newType) {
+		org.eclipse.core.runtime.Assert.isNotNull(newType, "DataType of a Events must not be null! " + getName());
```

Comment:
I would suggest instead using the more ubiquitous:
```suggestion
		java.util.Objects.requireNonNull(newType, "DataType of a Events must not be null! " + getName());
```

Reply:
done